File: resources/templates/js/Director.js
${NAME} = Class({

    Extends: DirectorScript,

    onStarted: function() {
    	// TODO Super will create a default stage and view. Omit this if you want to create your own stages and views instead.
    	Super();
    	// TODO Initialise your object.
    },
    
    tick: function() {
        // TODO Called 50 times a second.
        // Consider using a SceneDirector, rather than putting code here.
    }
    
    // Other methods include :
    // onActivate, onDeactivate, onQuit, onKeyDown, onKeyUp, onMouseDown, onMouseUp, onMouseMove, onMessage
    // Note : SceneDirector is often a better place to handle mouse and key events.
});

// NOTE. You can access this object from your other scripts using the global variable : 'directorScript'.

File: resources/templates/py/Director.py
from uk.co.nickthecoder.itchy import AbstractDirector
from uk.co.nickthecoder.itchy.util import ClassName

from java.util import ArrayList

class ${Name}(AbstractDirector) :

    def __init__(self) :
        # TODO Initialise your object.
        pass

    def tick(self) :
        # TODO Called 60 times a second.
        # It is often better to put game logic in SceneDirector subclasses, rather than Director subclasses.
        pass
    
    # TODO Other methods include :
    # onStarted, onMouseDown, onMouseUp, onMouseMove, onKeyDown, onKeyUp, onQuit, onMessage

    # Boiler plate code - no need to change this
    def getClassName(self):
        return ClassName( CostumeProperties, self.__module__ + ".py" )


